prerender/app.py

```diff
@@ -135,16 +137,32 @@ def _save_to_cache(key: str, data: bytes, format: str = 'html') -> None:
     if not skip_cache:
         try:
             data = await cache.get(url, format)
+            modified_since = await cache.modified_since(url)
```

Comment:
Maybe if `None` then just return current time? We can't check this if `skip_cache` is set.